diff --git a/lib/index.js b/lib/index.js
index f4eed2c5806cd8deca710870140609d27e895aa9..f137c650aa0dad18e1263191f52a16e505d62eb0 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -38,7 +38,7 @@ function _validate(format) {
 }
 
 function _clean(format) {
-  const expectedKeys = ['label', 'message', 'labelColor', 'color', 'style']
+  const expectedKeys = ['label', 'message', 'labelColor', 'color', 'style', 'logo']
 
   const cleaned = {}
   Object.keys(format).forEach(key => {